### PR TITLE
chore(dependencies): Remove react imports from ts files

### PR DESCRIPTION
**Pull request by toncabral88**
## Description
Removers all react imports from project and updates script to generate new component

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Code review
- [ ] TypeScript updated

### CODE REVIEW

**Review by matheusrpp — APPROVED** [review]
## Review guide
- [x] Code review
- [x] TypeScript updated

File: scripts/templates/component.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 module.exports = componentName => [
   {
     content: `// Generated with scripts/create-component.js
-import React from 'react';
 import PropTypes from 'prop-types';
 
 const ${componentName} = ({ someProp }) => (
```
Comment (rapahaeru) [inline scripts/templates/component.js:6]: in line 9, the text is wrong.
`<h1>Hello, do domething here!</h1>`

File: scripts/templates/component-story.js [hunk 2]
```diff
@@ -1,7 +1,6 @@
 module.exports = componentName => [
   {
     content: `// Generated with scripts/create-componnent.js
-import React from 'react';
 import {
   AutoExample,
```
Comment (rapahaeru) [inline scripts/templates/component-story.js:5]: Shouldnt this story be updated in new template of docs from storybook?
In this example, the storybook-ui still in use.

**Review by rapahaeru — CHANGES_REQUESTED** [review]

**Comment by rapahaeru** [comment]
## Review guide
- [ ] Code review
- [x] TypeScript updated

**Review by rapahaeru — APPROVED** [review]

**Review by matheusrpp — APPROVED** [review]

**Pull request merged**